SubmissionExample.ipynb

```diff
@@ -121,17 +111,17 @@
     "\n",
     "# mobilenet\n",
```

Comment:
we should take mobilenet out since it is gone since v0.7

Reply:
I was not aware.  @christ1ne - is it alright to remove mobilenet entirely?

Reply:
Christine confirmed offline.  I will remove this and other references to MobileNet.